docs/kyma/04-04-cluster-installation.md

```diff
@@ -39,7 +39,7 @@ This installation guide explains how you can quickly deploy Kyma on a cluster wi
   Gardener
   </summary>
 
-- [Gardener](https://gardener.cloud/) seed cluster
+- [Gardener](https://gardener.cloud/) account
 - [Google Cloud Platform](https://console.cloud.google.com/) (GCP) project with Kubernetes Engine API enabled or a [Microsoft Azure](https://azure.microsoft.com) account
```

Comment:
For Gardener on GCP we need a GCP project, probably having the Compute Engine API enabled but we can skip that, definitly not the kubernetes Engine API.
For Gardener on Azure we need an Azure project

Reply:
Changed :)